pkg/cloud-init/cloud-init.go

```diff
@@ -465,6 +466,56 @@ func removeLocalData(domain string, namespace string) error {
 	return err
 }
 
+func GenerateEmptyIso(vmiName string, namespace string, data *CloudInitData, sizes *v1.VirtualMachineInstanceIsoSizes) error {
```

Comment:
I think we need a basic unit test at least to make sure this function does what we want

Reply:
@vladikr pushed!